src/sentry/static/sentry/less/type.less

```diff
@@ -181,6 +184,16 @@ pre {
   word-break: keep-all;
   word-wrap: normal;
   padding: 10px;
+
+  .script {
+    color: @script-blue;
+  }
+  .attribute {
+    color: @attribute-purple;
+  }
+  .value-text {
```

Comment:
these are very globally scoped - the safe thing to do would be to use a much more specific css rule (in a different file, since this is a temporary experiment) or just emotion and styled components in the javascript file. you can also hardcode these color values there or define them more locally

Reply:
Good point! I'll use styled components instead